Jenkinsfile

```diff
@@ -88,20 +88,6 @@ pipeline {
             }
         }
 
-        stage('Integration Tests Only Build') {
-            options {
-                timeout(time: 1, unit: 'HOURS')
-            }
-            steps {
-                withMaven(maven: 'maven-latest', jdk: 'jdk11', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
-                    sh '''
-                        unset JAVA_TOOL_OPTIONS
-                        mvn install -B -pl $ITESTS,!$ITCORE -amd -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
-                    '''
-                }
-            }
-        }
-
         stage('DDF Core Tests Only Build') {
```

Comment:
Should we remove all itests from DDF's remote builds, or should we keep the core itests in?

Reply:
seems core tests run in about 6 minutes, so it is not doing serious damage to build time. iTests add 45mins to an hour so that was a big driver for pulling it out to a separate build, at least that was my impression

Reply:
We want to keep the core itests are part of the DDF repo still. They were built to be a replacement for the large itests we are pulling out.